components/Layout.js

```diff
@@ -0,0 +1,12 @@
+import Footer from './Footer';
+import Header from './Header';
+
+export default function Layout({ children }) {
+  return (
+    <>
+      <Header />
+      {children}
```

Comment:
```suggestion
     <main>{children}</main>
```
the children should be wrapped in an html element as well. :) main works best!

Reply:
Done